shopelectro/views/ecommerce.py

```diff
@@ -34,6 +36,37 @@ def get_context_data(self, request, **kwargs):
         }
 
 
+# @todo #789:60m Make cart routes REST style instead of RPC.
+
+# @todo #789:15m Test http cache for cart-get route.
```

Comment:
In this case cache should be disabled, as i understand.
Clarify it plz in subtask body

Reply:
I will just drop this with cache condition logic